_includes/specials/taproot/en/18-trivia.md

```diff
@@ -0,0 +1,83 @@
+{% auto_anchor %}
+
+- **What is a taproot?** Wikipedia [says][wikipedia taproot], "A taproot
+  is a large, central, and dominant root from which other roots sprout
+  laterally. Typically a taproot is somewhat straight and very thick, is
+  tapering in shape, and grows directly downward.  In some plants, such
+  as the carrot, the taproot is a storage organ so well developed that
+  it has been cultivated as a vegetable."
+
+    Bitcoin's [taproot][topic taproot], similar to the plant root, hides
+    the details of its scripts below the surface, away from public view,
+    and is very efficient at storing those scripts.
+
+- **Schnorr predates ECDSA:** we talk about [schnorr signatures][topic
+  schnorr signatures] as an upgrade on Bitcoin's original ECDSA
+  signatures because they make it easier to implement various
+  cryptographic tricks, but the schnorr signature algorithm predates the
+  DSA algorithm that ECDSA is based upon.  Indeed, DSA was created in
+  part to circumvent the [patent on schnorr signatures][schnorr patent]
+  and it was [reported][gov report] that "there were allegations of
+  infringement from Mr. Claus Schnorr and Public Key Partners (PKP)"
+  that DSA violated Schnorr's patent.  That patent expired in 2008.
+
+- **Un-Schnorr what name to use:** although it didn't work out, there
+  was a [suggestion][dryja bn sigs] early in the development of adapting
+  schnorr signatures for Bitcoin that Claus Schnorr's name shouldn't be
+  used in association with them because his patent on them prevented the
+  widespread use of a valuable cryptographic technique for over 20
+  years.
+
+- **Schnorr for twisted Edwards curves:** an application of schnorr
+  signatures using elliptic curves was written in 2011 <!-- guessing
+  from URL; Wikipedia also says this without a source --> and
+  [published][ed25519] in 2012<!-- verified -->.  The scheme,
+  [EdDSA][], is now the basis of several standards.  Although not used
+  in Bitcoin consensus, references to it in the context of other systems
+  can be found in many of the Bitcoin repositories tracked by Optech.
```

Comment:
This could say that EdDSA influenced the design of BIP340 heavily. Without this, it's also not entirely clear to me what the purpose of the paragraph is.

Reply:
Just commenting on the history here.

It's just showing that eddsa (which is similar to bip340) is widely used.  I don't agree that the design of eddsa was particularly influential.  Or rather, it was a influence but only in the sense that the criteria for what signatures will be accepted/rejected is severely under-specified, leading to actual attacks in cryptocurrencies that use it... and BIP340 took care to not make the same error.  Otherwise, except for the tagged hashes (which eddsa doesn't use either) I believe BIP340 is compatible with the schnorr implementation that we created before we were really looking at eddsa at all.

Reply:
The old schnorr signature module in libsecp256k1 didn't have key prefixing (while eddsa does). I think we were aware of eddsa and key prefixing at the time, but it wasn't considered as essential. Only with MuSig's security proof relying on key prefixing, and the key malleability the paper highlighted, did we decide to standardize on key prefixing.

I'd say that BIP340 isn't exactly based on EdDSA, but the end result is rather similar. The fact that another reasonably similar Schnorr-based design existed probably helped being confident about it.

Reply:
Speaking of EdDSA the story comes to mind when we were specifying the nonce derivation function and quickly realized that for a key-prefixed scheme the public key should be input to the function to protect a user against accidental misuse. @gmaxwell [noticed that this is not the case](https://moderncrypto.org/mail-archive/curves/2020/001012.html) in EdDSA.

Reply:
Generally in English when we say that X was influenced by Y we don't *usually* mean by Y's errors and omissions. :P  Key prefixing is a true similarity but inside bitcoin before sighash no input (and likewise, e.g. sighash bug) were considered every transaction signature was inherently implicitly key-prefixed.